<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 6 Research Platforms | Computing for Cancer Informatics</title>
  <meta name="description" content="This course covers material about economic, privacy, security, usability, and discoverability aspects of data management for cancer research." />
  <meta name="generator" content="bookdown 0.41 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 6 Research Platforms | Computing for Cancer Informatics" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="This course covers material about economic, privacy, security, usability, and discoverability aspects of data management for cancer research." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 6 Research Platforms | Computing for Cancer Informatics" />
  
  <meta name="twitter:description" content="This course covers material about economic, privacy, security, usability, and discoverability aspects of data management for cancer research." />
  



<meta name="date" content="2025-06-06" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="assets/ITN_favicon.ico" type="image/x-icon" />
<link rel="prev" href="shared-computing-etiquette.html"/>
<link rel="next" href="computing-resource-decisions.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>
<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="assets/style_ITN.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<a href="https://www.itcrtraining.org/"><img src="assets/ITN_logo.png" style="padding-left: 15px; padding-top: 8px;"</a>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About This Course</a>
<ul>
<li class="chapter" data-level="0.1" data-path="index.html"><a href="index.html#available-course-formats"><i class="fa fa-check"></i><b>0.1</b> Available course formats</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#motivation"><i class="fa fa-check"></i><b>1.1</b> Motivation</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#target-audience"><i class="fa fa-check"></i><b>1.2</b> Target Audience</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#topics-covered"><i class="fa fa-check"></i><b>1.3</b> Topics covered:</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#curriculum"><i class="fa fa-check"></i><b>1.4</b> Curriculum</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="basic-building-block-of-computers.html"><a href="basic-building-block-of-computers.html"><i class="fa fa-check"></i><b>2</b> Basic Building Block of Computers</a>
<ul>
<li class="chapter" data-level="2.1" data-path="basic-building-block-of-computers.html"><a href="basic-building-block-of-computers.html#transistors"><i class="fa fa-check"></i><b>2.1</b> Transistors</a></li>
<li class="chapter" data-level="2.2" data-path="basic-building-block-of-computers.html"><a href="basic-building-block-of-computers.html#alu---arithmetic-logic-unit"><i class="fa fa-check"></i><b>2.2</b> ALU - Arithmetic Logic Unit</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="basic-building-block-of-computers.html"><a href="basic-building-block-of-computers.html#binary-data"><i class="fa fa-check"></i><b>2.2.1</b> Binary data</a></li>
<li class="chapter" data-level="2.2.2" data-path="basic-building-block-of-computers.html"><a href="basic-building-block-of-computers.html#flip-flops-and-registers"><i class="fa fa-check"></i><b>2.2.2</b> Flip-flops and registers</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="basic-building-block-of-computers.html"><a href="basic-building-block-of-computers.html#conclusion"><i class="fa fa-check"></i><b>2.3</b> Conclusion</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="binary-data-to-computations.html"><a href="binary-data-to-computations.html"><i class="fa fa-check"></i><b>3</b> Binary Data to Computations</a>
<ul>
<li class="chapter" data-level="3.0.1" data-path="binary-data-to-computations.html"><a href="binary-data-to-computations.html#cpu---central-processing-unit"><i class="fa fa-check"></i><b>3.0.1</b> <strong>CPU</strong> - Central Processing Unit</a></li>
<li class="chapter" data-level="3.0.2" data-path="binary-data-to-computations.html"><a href="binary-data-to-computations.html#memory-or-ram---short-term-memory"><i class="fa fa-check"></i><b>3.0.2</b> <strong>Memory or RAM</strong> - short-term memory</a></li>
<li class="chapter" data-level="3.0.3" data-path="binary-data-to-computations.html"><a href="binary-data-to-computations.html#storage---long-term-memory"><i class="fa fa-check"></i><b>3.0.3</b> <strong>Storage</strong> - long-term memory</a></li>
<li class="chapter" data-level="3.0.4" data-path="binary-data-to-computations.html"><a href="binary-data-to-computations.html#hardware-and-software"><i class="fa fa-check"></i><b>3.0.4</b> Hardware and software</a></li>
<li class="chapter" data-level="3.0.5" data-path="binary-data-to-computations.html"><a href="binary-data-to-computations.html#operating-systems"><i class="fa fa-check"></i><b>3.0.5</b> Operating systems</a></li>
<li class="chapter" data-level="3.0.6" data-path="binary-data-to-computations.html"><a href="binary-data-to-computations.html#historical-context"><i class="fa fa-check"></i><b>3.0.6</b> Historical context</a></li>
<li class="chapter" data-level="3.1" data-path="binary-data-to-computations.html"><a href="binary-data-to-computations.html#conclusion-1"><i class="fa fa-check"></i><b>3.1</b> Conclusion</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="computing-resources.html"><a href="computing-resources.html"><i class="fa fa-check"></i><b>4</b> Computing Resources</a>
<ul>
<li class="chapter" data-level="4.1" data-path="computing-resources.html"><a href="computing-resources.html#data-sizes"><i class="fa fa-check"></i><b>4.1</b> Data Sizes</a></li>
<li class="chapter" data-level="4.2" data-path="computing-resources.html"><a href="computing-resources.html#computing-capacity"><i class="fa fa-check"></i><b>4.2</b> Computing Capacity</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="computing-resources.html"><a href="computing-resources.html#checking-your-computer-capacity---mac"><i class="fa fa-check"></i><b>4.2.1</b> Checking your computer capacity - Mac</a></li>
<li class="chapter" data-level="4.2.2" data-path="computing-resources.html"><a href="computing-resources.html#checking-your-computer-capacity---windowspc"><i class="fa fa-check"></i><b>4.2.2</b> Checking your computer capacity - Windows/PC</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="computing-resources.html"><a href="computing-resources.html#file-sizes"><i class="fa fa-check"></i><b>4.3</b> File Sizes</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="computing-resources.html"><a href="computing-resources.html#genomic-data-file-sizes"><i class="fa fa-check"></i><b>4.3.1</b> Genomic data file sizes</a></li>
<li class="chapter" data-level="4.3.2" data-path="computing-resources.html"><a href="computing-resources.html#imaging-data-file-sizes"><i class="fa fa-check"></i><b>4.3.2</b> Imaging Data File Sizes</a></li>
<li class="chapter" data-level="4.3.3" data-path="computing-resources.html"><a href="computing-resources.html#clinical-data-file-sizes"><i class="fa fa-check"></i><b>4.3.3</b> Clinical Data File Sizes</a></li>
<li class="chapter" data-level="4.3.4" data-path="computing-resources.html"><a href="computing-resources.html#checking-file-sizes-on-mac"><i class="fa fa-check"></i><b>4.3.4</b> Checking file sizes on Mac</a></li>
<li class="chapter" data-level="4.3.5" data-path="computing-resources.html"><a href="computing-resources.html#checking-file-sizes-on-pcwindows"><i class="fa fa-check"></i><b>4.3.5</b> Checking file sizes on PC/Windows</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="computing-resources.html"><a href="computing-resources.html#computing-options"><i class="fa fa-check"></i><b>4.4</b> Computing Options</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="computing-resources.html"><a href="computing-resources.html#personal-computers"><i class="fa fa-check"></i><b>4.4.1</b> <strong>Personal computers</strong></a></li>
<li class="chapter" data-level="4.4.2" data-path="computing-resources.html"><a href="computing-resources.html#shared-computing-resources"><i class="fa fa-check"></i><b>4.4.2</b> <strong>Shared Computing Resources</strong></a></li>
<li class="chapter" data-level="4.4.3" data-path="computing-resources.html"><a href="computing-resources.html#computer-cluster"><i class="fa fa-check"></i><b>4.4.3</b> Computer Cluster</a></li>
<li class="chapter" data-level="4.4.4" data-path="computing-resources.html"><a href="computing-resources.html#computer-grid"><i class="fa fa-check"></i><b>4.4.4</b> Computer Grid</a></li>
<li class="chapter" data-level="4.4.5" data-path="computing-resources.html"><a href="computing-resources.html#cloud-computing"><i class="fa fa-check"></i><b>4.4.5</b> “Cloud” computing</a></li>
<li class="chapter" data-level="4.4.6" data-path="computing-resources.html"><a href="computing-resources.html#accessing-shared-computer-resources"><i class="fa fa-check"></i><b>4.4.6</b> Accessing Shared Computer Resources</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="computing-resources.html"><a href="computing-resources.html#conclusion-2"><i class="fa fa-check"></i><b>4.5</b> Conclusion</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="shared-computing-etiquette.html"><a href="shared-computing-etiquette.html"><i class="fa fa-check"></i><b>5</b> Shared Computing Etiquette</a>
<ul>
<li class="chapter" data-level="5.1" data-path="shared-computing-etiquette.html"><a href="shared-computing-etiquette.html#general-guidelines-for-shared-computing-resources"><i class="fa fa-check"></i><b>5.1</b> General Guidelines for shared computing resources</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="shared-computing-etiquette.html"><a href="shared-computing-etiquette.html#security-guidelines"><i class="fa fa-check"></i><b>5.1.1</b> Security guidelines</a></li>
<li class="chapter" data-level="5.1.2" data-path="shared-computing-etiquette.html"><a href="shared-computing-etiquette.html#overall-use-guidelines"><i class="fa fa-check"></i><b>5.1.2</b> Overall use guidelines</a></li>
<li class="chapter" data-level="5.1.3" data-path="shared-computing-etiquette.html"><a href="shared-computing-etiquette.html#daily-use-guidelines"><i class="fa fa-check"></i><b>5.1.3</b> Daily use guidelines</a></li>
<li class="chapter" data-level="5.1.4" data-path="shared-computing-etiquette.html"><a href="shared-computing-etiquette.html#communication-guidelines"><i class="fa fa-check"></i><b>5.1.4</b> Communication Guidelines</a></li>
<li class="chapter" data-level="5.1.5" data-path="shared-computing-etiquette.html"><a href="shared-computing-etiquette.html#specific-rules"><i class="fa fa-check"></i><b>5.1.5</b> Specific Rules</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="shared-computing-etiquette.html"><a href="shared-computing-etiquette.html#interacting-with-shared-resources"><i class="fa fa-check"></i><b>5.2</b> Interacting with shared resources</a></li>
<li class="chapter" data-level="5.3" data-path="shared-computing-etiquette.html"><a href="shared-computing-etiquette.html#running-jobs"><i class="fa fa-check"></i><b>5.3</b> Running Jobs</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="shared-computing-etiquette.html"><a href="shared-computing-etiquette.html#specifying-memory-ram-needs"><i class="fa fa-check"></i><b>5.3.1</b> Specifying memory (RAM) needs</a></li>
<li class="chapter" data-level="5.3.2" data-path="shared-computing-etiquette.html"><a href="shared-computing-etiquette.html#checking-status"><i class="fa fa-check"></i><b>5.3.2</b> Checking status</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="shared-computing-etiquette.html"><a href="shared-computing-etiquette.html#storage"><i class="fa fa-check"></i><b>5.4</b> Storage</a></li>
<li class="chapter" data-level="5.5" data-path="shared-computing-etiquette.html"><a href="shared-computing-etiquette.html#conclusion-3"><i class="fa fa-check"></i><b>5.5</b> Conclusion</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="research-platforms.html"><a href="research-platforms.html"><i class="fa fa-check"></i><b>6</b> Research Platforms</a>
<ul>
<li class="chapter" data-level="6.0.1" data-path="research-platforms.html"><a href="research-platforms.html#national-cancer-institute-cloud-resources"><i class="fa fa-check"></i><b>6.0.1</b> National Cancer Institute Cloud Resources</a></li>
<li class="chapter" data-level="6.0.2" data-path="research-platforms.html"><a href="research-platforms.html#cancer-genomics-cloud"><i class="fa fa-check"></i><b>6.0.2</b> Cancer Genomics Cloud</a></li>
<li class="chapter" data-level="6.0.3" data-path="research-platforms.html"><a href="research-platforms.html#institute-for-systems-biology-isb-cancer-gateway-in-the-cloud"><i class="fa fa-check"></i><b>6.0.3</b> Institute for Systems Biology (ISB) Cancer Gateway in the Cloud</a></li>
<li class="chapter" data-level="6.0.4" data-path="research-platforms.html"><a href="research-platforms.html#broad-institute-firecloud"><i class="fa fa-check"></i><b>6.0.4</b> Broad Institute FireCloud</a></li>
<li class="chapter" data-level="6.0.5" data-path="research-platforms.html"><a href="research-platforms.html#galaxy"><i class="fa fa-check"></i><b>6.0.5</b> Galaxy</a></li>
<li class="chapter" data-level="6.0.6" data-path="research-platforms.html"><a href="research-platforms.html#terra"><i class="fa fa-check"></i><b>6.0.6</b> Terra</a></li>
<li class="chapter" data-level="6.1" data-path="research-platforms.html"><a href="research-platforms.html#anvil"><i class="fa fa-check"></i><b>6.1</b> AnVIL</a></li>
<li class="chapter" data-level="6.2" data-path="research-platforms.html"><a href="research-platforms.html#cyverse"><i class="fa fa-check"></i><b>6.2</b> CyVerse</a></li>
<li class="chapter" data-level="6.3" data-path="research-platforms.html"><a href="research-platforms.html#sciserver"><i class="fa fa-check"></i><b>6.3</b> SciServer</a></li>
<li class="chapter" data-level="6.4" data-path="research-platforms.html"><a href="research-platforms.html#materials-cloud"><i class="fa fa-check"></i><b>6.4</b> Materials Cloud</a></li>
<li class="chapter" data-level="6.5" data-path="research-platforms.html"><a href="research-platforms.html#overture"><i class="fa fa-check"></i><b>6.5</b> Overture</a></li>
<li class="chapter" data-level="6.6" data-path="research-platforms.html"><a href="research-platforms.html#globus"><i class="fa fa-check"></i><b>6.6</b> Globus</a></li>
<li class="chapter" data-level="6.7" data-path="research-platforms.html"><a href="research-platforms.html#basespace-sequence-hub"><i class="fa fa-check"></i><b>6.7</b> BaseSpace Sequence Hub</a></li>
<li class="chapter" data-level="6.8" data-path="research-platforms.html"><a href="research-platforms.html#atlas.ti"><i class="fa fa-check"></i><b>6.8</b> ATLAS.ti</a></li>
<li class="chapter" data-level="6.9" data-path="research-platforms.html"><a href="research-platforms.html#genepattern"><i class="fa fa-check"></i><b>6.9</b> GenePattern</a></li>
<li class="chapter" data-level="6.10" data-path="research-platforms.html"><a href="research-platforms.html#xnat"><i class="fa fa-check"></i><b>6.10</b> XNAT</a></li>
<li class="chapter" data-level="6.11" data-path="research-platforms.html"><a href="research-platforms.html#ohif"><i class="fa fa-check"></i><b>6.11</b> OHIF</a></li>
<li class="chapter" data-level="6.12" data-path="research-platforms.html"><a href="research-platforms.html#prism"><i class="fa fa-check"></i><b>6.12</b> PRISM</a></li>
<li class="chapter" data-level="6.13" data-path="research-platforms.html"><a href="research-platforms.html#conclusion-4"><i class="fa fa-check"></i><b>6.13</b> Conclusion</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="computing-resource-decisions.html"><a href="computing-resource-decisions.html"><i class="fa fa-check"></i><b>7</b> Computing Resource Decisions</a>
<ul>
<li class="chapter" data-level="7.1" data-path="computing-resource-decisions.html"><a href="computing-resource-decisions.html#general-computing-considerations"><i class="fa fa-check"></i><b>7.1</b> General Computing Considerations</a>
<ul>
<li class="chapter" data-level="7.1.1" data-path="computing-resource-decisions.html"><a href="computing-resource-decisions.html#computation-needs"><i class="fa fa-check"></i><b>7.1.1</b> Computation needs</a></li>
<li class="chapter" data-level="7.1.2" data-path="computing-resource-decisions.html"><a href="computing-resource-decisions.html#data-storage"><i class="fa fa-check"></i><b>7.1.2</b> Data storage</a></li>
<li class="chapter" data-level="7.1.3" data-path="computing-resource-decisions.html"><a href="computing-resource-decisions.html#multi-institute-collaboration"><i class="fa fa-check"></i><b>7.1.3</b> Multi-institute collaboration</a></li>
<li class="chapter" data-level="7.1.4" data-path="computing-resource-decisions.html"><a href="computing-resource-decisions.html#protected-data"><i class="fa fa-check"></i><b>7.1.4</b> Protected data</a></li>
<li class="chapter" data-level="7.1.5" data-path="computing-resource-decisions.html"><a href="computing-resource-decisions.html#costs"><i class="fa fa-check"></i><b>7.1.5</b> Costs</a></li>
<li class="chapter" data-level="7.1.6" data-path="computing-resource-decisions.html"><a href="computing-resource-decisions.html#extra-guidance"><i class="fa fa-check"></i><b>7.1.6</b> Extra guidance</a></li>
<li class="chapter" data-level="7.1.7" data-path="computing-resource-decisions.html"><a href="computing-resource-decisions.html#flexibility"><i class="fa fa-check"></i><b>7.1.7</b> Flexibility</a></li>
<li class="chapter" data-level="7.1.8" data-path="computing-resource-decisions.html"><a href="computing-resource-decisions.html#scalability"><i class="fa fa-check"></i><b>7.1.8</b> Scalability</a></li>
<li class="chapter" data-level="7.1.9" data-path="computing-resource-decisions.html"><a href="computing-resource-decisions.html#data-access"><i class="fa fa-check"></i><b>7.1.9</b> Data access</a></li>
<li class="chapter" data-level="7.1.10" data-path="computing-resource-decisions.html"><a href="computing-resource-decisions.html#interface"><i class="fa fa-check"></i><b>7.1.10</b> Interface</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="computing-resource-decisions.html"><a href="computing-resource-decisions.html#local-shared-resources-vs-remote-shared-resources"><i class="fa fa-check"></i><b>7.2</b> Local shared resources vs remote shared resources</a>
<ul>
<li class="chapter" data-level="7.2.1" data-path="computing-resource-decisions.html"><a href="computing-resource-decisions.html#benefits-of-cloud-computing"><i class="fa fa-check"></i><b>7.2.1</b> Benefits of Cloud Computing</a></li>
<li class="chapter" data-level="7.2.2" data-path="computing-resource-decisions.html"><a href="computing-resource-decisions.html#challenges-of-cloud-computing"><i class="fa fa-check"></i><b>7.2.2</b> Challenges of Cloud Computing</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="computing-resource-decisions.html"><a href="computing-resource-decisions.html#choosing-between-remote-sharing-options"><i class="fa fa-check"></i><b>7.3</b> Choosing between remote sharing options</a></li>
<li class="chapter" data-level="7.4" data-path="computing-resource-decisions.html"><a href="computing-resource-decisions.html#overall-decision-process"><i class="fa fa-check"></i><b>7.4</b> Overall Decision Process</a></li>
<li class="chapter" data-level="7.5" data-path="computing-resource-decisions.html"><a href="computing-resource-decisions.html#conclusion-5"><i class="fa fa-check"></i><b>7.5</b> Conclusion</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="about-the-authors.html"><a href="about-the-authors.html"><i class="fa fa-check"></i>About the Authors</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<p style="text-align:center;"> <a href="https://github.com/jhudsl/Computing_for_Cancer_Informatics" target="blank" > This content was published with</a> <a href="https://bookdown.org/" target="blank"> bookdown by: </a> </p>
<p style="text-align:center;"> <a href="http://jhudatascience.org/"> The Johns Hopkins Data Science Lab </a></p>
<a href="http://jhudatascience.org/"><img src="https://jhudatascience.org/images/dasl.png" style=" width: 80%; filter: grayscale(100%); padding-left: 40px; padding-top: 8px; vertical-align: top "</a>
<p style="text-align:center; font-size: 12px;"> <a href="https://github.com/rstudio4edu/rstudio4edu-book/"> Style adapted from: rstudio4edu-book </a> <a href ="https://creativecommons.org/licenses/by/2.0/"> (CC-BY 2.0) </a></p>
<p style="padding-left: 40px;"><div class="trapezoid" style = "padding-left: 40px;"><span>  <a href="https://forms.gle/hc8Xt3Y2Znjb6M4Y7"> Click here to provide feedback</a> <img src="assets/itcr_arrow.png" style=" width: 10%" ></span></div></p>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Computing for Cancer Informatics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
  <html>
  
  <head>
  <title>Title</title>
  </head>
  
  <body>
  
  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-TGSJ7H9TXE"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());
    
    gtag('config', 'G-TGSJ7H9TXE');
  </script>
      
  </body>
  </html>
<head>
  <meta name="viewport" content="width=device-width,minimum-scale=1.0,maximum-scale=1.0,initial-scale=1.0">
  <!--script src="https://kit.fontawesome.com/6a26f47516.js"></script-->
  <!--<script src="assets/hideOutput.js"></script>-->
  <link href="assets/style_ITN.css" rel="stylesheet">
</head>
        


<div class="hero-image-container"> 
  <img class= "hero-image" src="assets/itcr_main_image.png">
</div>
<div id="research-platforms" class="section level1 hasAnchor" number="6">
<h1><span class="header-section-number">Chapter 6</span> Research Platforms<a href="research-platforms.html#research-platforms" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>In this chapter, we will provide examples of computing platforms designed to help researchers. You might find these platforms useful for your work. Please note that we aim to provide a general overview of options, and thus, this is not a complete list. Let us know if there is a platform or system that you think we should include!</p>
<div class="warning">
<p>We highly suggest you also <strong>read the next chapter</strong>, which will point out important considerations to think about when deciding to work on a shared computing resource platform, like those discussed in this chapter.</p>
</div>
<p><img src="resources/images/06-General_Platforms_files/figure-html/1B4LwuvgA6aUopOHEAbES1Agjy7Ex2IpVAoUIoBFbsq0_gf5f8818810_1_171.png" alt="Learning Objectives: 1.Recognize general informatics computing and data management platforms. 2.Be aware of differnt benefits of the various platforms" width="100%" style="display: block; margin: auto;" /></p>
<p>The major advantage of these platforms is that users can analyze data where it lives, as many platforms host public data. However, some also allow you to upload your own data. There is less need for data transfers back and forth to your personal computer, as you can analyze your data, store your data and share it in one place, saving time. Users can sometimes also share how they did their analysis as well, improving reproducibility practices. Additionally, another advantage is that some of these platforms also provide educational material on how to work with data.</p>
<p>Many offer a <a href="https://www.omnisci.com/technical-glossary/graphical-user-interface">graphical user interface</a> also simply called just graphical interface or GUI (side note: GUI is pronounced like the word “gooey”, as if it’s a sticky jelly stuck to the monitor!), allows for users to choose functions to perform by interacting with visual representations, which can be useful for individuals how are less comfortable writing code. They have a “user-centered” design that creates a visual environment where users can for example <strong>click on</strong> tabs, boxes, or icons for to perform functions. This also often allows users to more directly see plots and other types of visualizations.</p>
<p><img src="resources/images/06-General_Platforms_files/figure-html/1B4LwuvgA6aUopOHEAbES1Agjy7Ex2IpVAoUIoBFbsq0_g11bbe6ab7c6_0_10.png" alt="Think of the GUI (i.e. gooey) as the sticky goo that 'sticks' to the monitor, and helps you navigate your interactions with the computer!" width="100%" style="display: block; margin: auto;" /></p>
<p>Some platforms also offer a <a href="https://searchwindowsserver.techtarget.com/definition/command-line-interface-CLI">command line interface</a> (also known as a character interface) which allows for software functions to be performed by specifying through commands written in text. This typically offers more control than a GUI; however, command line interfaces are often less user-friendly as they require that the user know the correct commands to use.</p>
<div id="national-cancer-institute-cloud-resources" class="section level3 hasAnchor" number="6.0.1">
<h3><span class="header-section-number">6.0.1</span> National Cancer Institute Cloud Resources<a href="research-platforms.html#national-cancer-institute-cloud-resources" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Funded by the <a href="https://www.cancer.gov/">National Cancer Institute (NCI)</a>, the <a href="https://datascience.cancer.gov/data-commons">cancer research data commons</a> provides data access and computing infrastructure for researchers through <a href="https://dcf.gen3.org/cloud-resources">three different platforms</a>, the <a href="https://www.cancergenomicscloud.org/">Cancer Genomics Cloud (CGC)</a> which uses Google Cloud resources, the <a href="https://isb-cgc.appspot.com/">Institute for Systems Biology Cancer Gateway in the Cloud (ISB-CGC)</a> which also uses Google Cloud resources, and <a href="https://portal.firecloud.org/">FireCloud</a> from the Broad Institute, which uses Amazon Cloud resources.</p>
</div>
<div id="cancer-genomics-cloud" class="section level3 hasAnchor" number="6.0.2">
<h3><span class="header-section-number">6.0.2</span> Cancer Genomics Cloud<a href="research-platforms.html#cancer-genomics-cloud" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The <a href="https://www.cancergenomicscloud.org/">Cancer Genomics Cloud (CGC)</a> is a computing platform that researchers can use to analyze, store, and share their own data, as well as work with large public and controlled cancer data sets, including genomic and imaging data. CGC offers tutorials and guides to help research get started, as well as $300 of free credits to use the platform and test it out. Users can also access many tools and workflows to help them perform there analyses. CGC also offers regular <a href="https://www.cancergenomicscloud.org/webinars">webinars</a>.</p>
<p>The platform is based on a partnership with <a href="https://www.sevenbridges.com/">Seven Bridges</a>, a biomedical analytics company, and can be accessed simply by using a web browser. Users can use a point-and-click system (GUI) or access resources using the command line. See this <a href="https://www.cancergenomicscloud.org/getting-started">link</a> to learn more.</p>
</div>
<div id="institute-for-systems-biology-isb-cancer-gateway-in-the-cloud" class="section level3 hasAnchor" number="6.0.3">
<h3><span class="header-section-number">6.0.3</span> Institute for Systems Biology (ISB) Cancer Gateway in the Cloud<a href="research-platforms.html#institute-for-systems-biology-isb-cancer-gateway-in-the-cloud" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The <a href="https://isb-cgc.appspot.com/">ISB-CRC</a> platform allows users to browse and data from the <a href="https://gdc.cancer.gov/">Genomic Data Commons</a> and other sources, including sequencing and imaging data that are both public and controlled. They provide access pipeline tools, as well as to pipelines, workflows, and Notebooks written by others in R and Python to help users perform analyses. ISB also offers $300 in <a href="https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/HowtoRequestCloudCredits.html">free credits</a> to try out the platform. See <a href="https://isb-cgc.appspot.com/how_to_discover/#0">here</a> for a user guide.</p>
</div>
<div id="broad-institute-firecloud" class="section level3 hasAnchor" number="6.0.4">
<h3><span class="header-section-number">6.0.4</span> Broad Institute FireCloud<a href="research-platforms.html#broad-institute-firecloud" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><a href="https://portal.firecloud.org/">FireCloud</a> provides users with computing resources and access to workspaces using Broad’s tools and pipelines. Users can run large scale analyses and work with collaborators. FireCloud offers access to <a href="https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga">The Cancer Genome Atlas (TCGA)</a> controlled-access data. Other platforms described next, like Galaxy and Terra, share resources with FireCloud.</p>
</div>
<div id="galaxy" class="section level3 hasAnchor" number="6.0.5">
<h3><span class="header-section-number">6.0.5</span> Galaxy<a href="research-platforms.html#galaxy" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>This section was written by <a href="https://www.goeckslab.org/members/jeremy-goecks.html">Jeremy Goecks</a>:</p>
<p>Galaxy is a web-based computational workbench that connects analysis tools, biomedical datasets, computing resources, a graphical user interface, and a programmatic API. Galaxy (<a href="https://galaxyproject.org/" class="uri">https://galaxyproject.org/</a>) enables accessible, reproducible, and collaborative biomedical data science regardless of a user’s informatics expertise. There are more than 8,000 analysis tools and 200 visualizations integrated into Galaxy that can be used to process a wide variety of biomedical datasets. This includes tools for analyzing genomic, transcriptomic (RNA-seq), proteomic, metabolomic, microbiome, and imaging datasets, tool suites for single-cell omics and machine learning, and thousands of more tools. Galaxy’s graphical user interface can be used simply through a web browser, and there is a programmatic API for performing scripted and automated analyses with Galaxy.</p>
<p><img src="resources/images/06-General_Platforms_files/figure-html/1B4LwuvgA6aUopOHEAbES1Agjy7Ex2IpVAoUIoBFbsq0_gfb2e21ecdc_0_131.png" alt="Galaxy can be accessed through a web browser and provides users with access to tools, datasets, computing resources, a graphical user interface (GUI) for users who would like to interact with Galaxy by clicking buttons and using drop-down menus and a programmtic API for users that would like to write code to interact with Galaxy" width="100%" style="display: block; margin: auto;" /></p>
<p>Galaxy is used daily by thousands of scientists across the world. A vibrant Galaxy community has deployed hundreds of Galaxy servers across the world, including more than 150 public and three large national/international servers in the United States, Europe, and Australia (<a href="https://usegalaxy.org" class="uri">https://usegalaxy.org</a>, <a href="https://usegalaxy.eu" class="uri">https://usegalaxy.eu</a>, <a href="https://usegalaxy.org.au" class="uri">https://usegalaxy.org.au</a>). The three national/international servers have more than 250,000 registered users who execute &gt;500,000 analysis jobs each month. Galaxy has been cited more than 10,000 times with &gt;20% from papers related to cancer. The Galaxy Tool Shed (<a href="https://usegalaxy.org/toolshed" class="uri">https://usegalaxy.org/toolshed</a>) provides a central location where developers can upload tools and visualizations, and users can search and install tools and visualizations into any Galaxy server. Galaxy has a large presence in the cancer research community. Galaxy serves as an integration and/or analysis platform for 7 projects in the NCI ITCR program. There is also increasing use of Galaxy in key NIH initiatives such as the NCI Cancer Moonshot Human Tumor Atlas Network (HTAN) and the NHGRI Data Commons, called the AnVIL (<a href="https://anvilproject.org/" class="uri">https://anvilproject.org/</a>).</p>
<p><img src="resources/images/06-General_Platforms_files/figure-html/1B4LwuvgA6aUopOHEAbES1Agjy7Ex2IpVAoUIoBFbsq0_gfb2e21ecdc_0_135.png" alt="Map of the 3 Galaxy servers" width="100%" style="display: block; margin: auto;" /></p>
<p>Galaxy’s user interface, accessible via a web browser, provides access to all Galaxy functionality. The main Galaxy interface has three panels: available tools (left), running analyses and viewing data (middle), and a full history of tools run and datasets generated (right).</p>
<p><img src="resources/images/06-General_Platforms_files/figure-html/1B4LwuvgA6aUopOHEAbES1Agjy7Ex2IpVAoUIoBFbsq0_gfb2e21ecdc_0_111.png" alt="Galaxy user interface showing the 3 panels: left = tools, middle = analysis and data, and right = history and generated datasets" width="100%" style="display: block; margin: auto;" /></p>
<p>Datasets for analysis in Galaxy can be <strong>uploaded</strong> from a laptop or desktop computer or obtained from public data repositories connected to Galaxy. With Galaxy, complex workflows composed of tens or even hundreds of analysis tools can be created and run. In Galaxy’s workflow interface, tools can be added and connected via a simple drag-and-drop approach.</p>
<p><img src="resources/images/06-General_Platforms_files/figure-html/1B4LwuvgA6aUopOHEAbES1Agjy7Ex2IpVAoUIoBFbsq0_gfb2e21ecdc_0_141.png" alt="Galaxy workflow example showing the beginning steps of a workflow" width="100%" style="display: block; margin: auto;" /></p>
<p>Galaxy users can share all their work—analysis histories, workflows, and visualizations—via simple URLs. They can be shared to specific colleagues or to anyone. Furthermore, Galaxy’s user interface is highly scalable. Tens, hundreds, or even thousands of datasets can be grouped into collections and run in parallel using individual tools or multi-tool workflows. In summary, Galaxy is a popular computational workbench with tools and features for a wide variety of data analyses, and it has broad usage in cancer data analysis.</p>
<p>See <a href="https://toolshed.g2.bx.psu.edu/">here</a> for the list of applications supported by Galaxy and <a href="https://training.galaxyproject.org/">here</a> for more information on how to use Galaxy resources.</p>
</div>
<div id="terra" class="section level3 hasAnchor" number="6.0.6">
<h3><span class="header-section-number">6.0.6</span> Terra<a href="research-platforms.html#terra" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><a href="https://terra.bio/">Terra</a> is a biomedical research computing platform that is based on the Google Cloud platform. Terra also allows users easier ways to manage the billing of their projects. It provides users with access to data, workflows, interactive analyses using Jupyter Notebooks, RStudio, and Galaxy, data access and tools from <a href="https://firecloud.terra.bio/">FireCloud from the Broad Institute</a>, as well as workspaces to organize projects and collaborate with others. Terra also has <a href="https://terra.bio/resources/security/">many measures</a> to help ensure that data are secure. They also offer clinical features to make sure <a href="https://terra.bio/about/privacy/">health data is protected</a>. Note that users who do upload protected health information must select to use extra clinical features and enter a formal agreement with <a href="https://firecloud.terra.bio/">Terra/FireCloud</a> about their data. See <a href="https://support.terra.bio/hc/en-us/articles/360024688731-Terms-of-Service">here</a> for more information.</p>
<p>Importantly users can get access to use <a href="https://gtexportal.org/home/">Genotype -Tissue Expression (GTEx)</a>, <a href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000218.v24.p8">Therapeutically Applicable Research to Generate Effective Treatments (TARGET)</a> and <a href="https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga">The Cancer Genome Atlas (TCGA)</a> data using the platform. See <a href="https://support.terra.bio/hc/en-us/articles/4402326091675-Accessing-GTEx-TARGET-TCGA-data">here</a> for information on how.</p>
<p>Users can pay for data storage and computing costs for Google Cloud through Terra. That said, browsing data is free.</p>
<p>Check out this video for more information:</p>
<iframe src="https://www.youtube.com/embed/3rH86vcAqK8" width="100%" height="400px" data-external="1">
</iframe>
</div>
<div id="anvil" class="section level2 hasAnchor" number="6.1">
<h2><span class="header-section-number">6.1</span> AnVIL<a href="research-platforms.html#anvil" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>If you could use some guidance on how to perform analyses using Galaxy and Terra, especially for genomic research, check out <a href="https://anvilproject.org/">AnVIL</a>, the <a href="https://www.genome.gov/">National Human Genome Research Institute (NHGRI)</a> Analysis Visualization and Informatics Lab-space. It also provides access to many important genomic and related <a href="https://anvilproject.org/data">datasets</a> from the NHGRI.</p>
<p>According to their website:</p>
<blockquote>
<p>By providing a unified environment for data management and compute, AnVIL eliminates the need for data movement, allows for active threat detection and monitoring, and provides elastic, shared computing resources that can be acquired by researchers as needed.</p>
</blockquote>
<p>AnVIL relies on Terra for the cloud based compute environment, Dockstore for standardized tools and workflows, Gen3 for data management for querying and organizing data, Galaxy tools and environment for analyses with less code requirements, and <a href="https://www.bioconductor.org/">Bioconductor</a> tools for R programming users.</p>
<p><a href="https://www.bioconductor.org/">Bioconductor</a> is a project with the mission to catalog, support, and disseminate bioinformatics open-source R packages. Packages have to go through a review process before being included.</p>
</div>
<div id="cyverse" class="section level2 hasAnchor" number="6.2">
<h2><span class="header-section-number">6.2</span> CyVerse<a href="research-platforms.html#cyverse" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p><a href="https://cyverse.rocks/about">CyVerse</a> is a similar computing platform that also offers computing resources for storing, sharing, and working with data with a graphical interface, as well as an API. Computing was previously offered using the cloud computing platform from CyVerse called <a href="https://cyverse.org/atmosphere">Atmosphere</a>, which relied on users using virtual machines. Users will now use a new version of Atmosphere with partnership with <a href="https://jetstream-cloud.org/">Jetstream</a>. This allows users to use containers for easier collaboration and also offers US users more computing power and storage. Originally called iPlant Collaborative, it was started through funding from the National Science Foundation (NSF) to support life sciences research, particularly to support ecology, biodiversity, sustainability, and agriculture research. It is led by the University of Arizona, the Texas Advanced Computing Center, and Cold Spring Harbor Laboratory. It offers access to an environment for performing analyses with Jupyter (for Python mostly) and RStudio (for R mostly) and a variety of tools for Genomic data analysis. See <a href="https://cyverse.atlassian.net/wiki/spaces/DEapps/pages/241882146/List+of+Applications">here</a> for a list of applications that are supported by CyVerse. Note that you can also install tools on both platforms. Both CyVerse and Galaxy offer lots of helpful documentation, to help users get started with informatics analyses.</p>
<p>See <a href="https://learning.cyverse.org/">here</a> to learn more.</p>
<p><img src="resources/images/06-General_Platforms_files/figure-html/1B4LwuvgA6aUopOHEAbES1Agjy7Ex2IpVAoUIoBFbsq0_gfd56752f25_0_0.png" alt="CyVerse graphical interface for performing analyses" width="100%" style="display: block; margin: auto;" /></p>
</div>
<div id="sciserver" class="section level2 hasAnchor" number="6.3">
<h2><span class="header-section-number">6.3</span> SciServer<a href="research-platforms.html#sciserver" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>SciServer is accessible through a web browser and allows users to store, upload, download, share, and work with data and common tools on the same platform. It was originally built for the astrophysics community (and called SkyServer) but it has now been adapted to be used by scientists of all fields and is indeed used by many in the genomics field. It allows users to use Python and R in environments like Jupyter notebooks and RStudio, and also supports (Structured Query Language) SQL for data querying and management and is built on the use of Docker.</p>
<p>The main idea of SciServer is based on this premise: “bring the analysis to the data”. It is free to use after users register. However, users can buy extra resources. Users can keep data private or share their data.</p>
<p>As compared to Galaxy, these resources may be better for users with a bit more familiarity with informatics but who require more flexibility. Specifically, these resources are ideal for working with collaborators such as physicists or material scientists, as there are more tools supported across disciplines. In addition, SciServer also gives users access to very large data sets on Petabyte-scale (note that some of these require special permission to use) and supports developers to create their own web interfaces called SciUIs for particular use cases.</p>
<p>For <span class="citation">Taghizadeh-Popp et al. (<a href="#ref-sciserver_2020">2020</a>)</span> for more information.</p>
<p><img src="resources/images/06-General_Platforms_files/figure-html/1B4LwuvgA6aUopOHEAbES1Agjy7Ex2IpVAoUIoBFbsq0_gfd56752f25_0_13.png" alt="The SciServer dashboard allows users to interact with the various resources and tools available through SciServer. Most are available through the command line but some have user interface options. Jupyter Notebooks, RStudio and SciUIs (interfaces created by other SciServer users) can be used to interactively explore and analyze data by users." width="100%" style="display: block; margin: auto;" /></p>
</div>
<div id="materials-cloud" class="section level2 hasAnchor" number="6.4">
<h2><span class="header-section-number">6.4</span> Materials Cloud<a href="research-platforms.html#materials-cloud" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Another resource that might be of interest to Python users, particularly those who collaborate with material scientists, is Materials Cloud. It is designed to promote reproducible work, collaboration, and sharing of resources among scientists, particularly for simulations for the materials science field. Users can share data in a citable way, download data, upload data, share workflows, and perform analyses.</p>
<p><img src="resources/images/06-General_Platforms_files/figure-html/1B4LwuvgA6aUopOHEAbES1Agjy7Ex2IpVAoUIoBFbsq0_gfd56752f25_0_6.png" alt="Materials Cloud resources are based on allowing users to Learn about resources, Work using the resources, Discover aspects about data that is available, Explore data with interactive graphs, and archive to store and share data." width="100%" style="display: block; margin: auto;" /></p>
<p>This resource uses <a href="https://www.materialscloud.org/work/aiidalab">AiiDAlab</a> as the computing environment for researchers, which is based on <a href="https://www.nature.com/articles/s41597-020-00638-4">AiiDA</a>. According to their website:</p>
<blockquote>
<p>AiiDAlab builds on <a href="https://www.nature.com/articles/s41597-020-00638-4">AiiDA</a> as the computational workflow engine, and the Jupyter environment (notebooks, widgets, …) for writing and sharing apps</p>
</blockquote>
<p>See <a href="https://www.sciencedirect.com/science/article/pii/S092702562030656X?via%3Dihub">here</a> to learn more about how AiiDAlab supports the sharing of scientific workflows, particularly for those that use Python.</p>
<p>To learn more about Materials Cloud, check out <span class="citation">Talirz et al. (<a href="#ref-talirz_materials_2020">2020</a>)</span>.</p>
</div>
<div id="overture" class="section level2 hasAnchor" number="6.5">
<h2><span class="header-section-number">6.5</span> Overture<a href="research-platforms.html#overture" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Overture is a relatively new option to perform large-scale genomic data analyses. You can upload, download, manage, analyze and share your data with authentication and authorization methods to add security. Although designed for genomic research, the <a href="https://www.overture.bio/documentation/dms/">data management system</a> can be used for other scientific domains. Currently, additional products are still being developed for analysis, visualization, and sharing. However, several collaborations have created new incredible resources using some of the existing and developing products that might be useful for your research. Alternatively, Overture has options to help you create your own platform - see <a href="https://www.overture.bio/services/">here</a> for more information. It is compatible with Google, Microsoft Azure, and PostgreSQL for storage options.</p>
<p>These collaborations using Overture products can be found on the <a href="https://www.overture.bio/case-studies/">case studies</a> page of the <a href="https://www.overture.bio/">Overture website</a>.</p>
<p>For example, the <a href="https://cancercollaboratory.org/">Cancer Genome Collaboratory</a> is one such collaboration. This is a cloud-based resource that allows researchers to perform analyses using <a href="https://en.wikipedia.org/wiki/International_Cancer_Genome_Consortium">International Cancer Genome Consortium (ICGC)</a> cancer genome data, which includes tumor mutation data from the <a href="https://en.wikipedia.org/wiki/The_Cancer_Genome_Atlas">The Cancer Genome Atlas (TCGA)</a> and the <a href="https://dcc.icgc.org/pcawg">Pan-Cancer Analysis of Whole Genomes (PCAWG)</a> mutation data. See <a href="https://cancercollaboratory.org/services-cloud-resources">here</a> for information about billing, storage capacity, access, and security.</p>
<p>In addition, Overture products have also been used to create other data resources, such as the <a href="https://portal.kidsfirstdrc.org/login">Kids First Data Resource Portal</a> which has childhood cancer and birth defect genomic data for over 76,000 samples, and the <a href="https://portal.gdc.cancer.gov/">National Cancer Institute’s Genomic Data Commons Data portal</a>, which also includes <a href="https://en.wikipedia.org/wiki/The_Cancer_Genome_Atlas">The Cancer Genome Atlas (TCGA)</a> and <a href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000218.v24.p8">Therapeutically Applicable Research to Generate Effective Treatments (TARGET)</a>. The portal supports some basic <a href="https://portal.gdc.cancer.gov/analysis">analyses</a> as well for clinical data statistics and survival analysis.</p>
</div>
<div id="globus" class="section level2 hasAnchor" number="6.6">
<h2><span class="header-section-number">6.6</span> Globus<a href="research-platforms.html#globus" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>This section was written by Brigitte Raumann:</p>
<p><a href="www.globus.org">Globus</a> (www.globus.org) is a cloud-hosted service for secure, reliable research data management that allows data movement, synchronization, sharing, and discovery. Users access Globus services via a <a href="https://app.globus.org/">web interface</a> or <a href="https://docs.globus.org/cli/">command line interface</a>. Developers can <a href="https://www.globus.org/platform">integrate Globus capabilities</a> into their research applications and <a href="https://docs.globus.org/modern-research-data-portal/">data portals</a>.</p>
<p>The <a href="https://www.globus.org/data-transfer">Globus Transfer</a> service provides ‘fire and forget’ high-performance data transfer and synchronization between storage system such as laptops, supercomputers, tape archives, HPC clusters, scientific instruments, as well as public cloud storage. Globus enables researchers to share their data without the need to create temporary collaborator accounts on local storage systems and without the need to copy data to an external file sharing service. This technology ensures that data movement and sharing of hundreds of terabytes of data, in some cases petabytes of data, can be done in a manner that ensures data confidentiality, minimizes demands on researchers’ time, and makes efficient use of available cyberinfrastructure. Transfer and sharing of <a href="https://www.globus.org/protected-data">protected data</a>, such as HIPAA-regulated data, is also supported. Globus can also <a href="https://www.globus.org/platform/services/flows">automate tasks</a> as simple as replicating data across multiple storage systems or as intricate as managing multiple conditional data analysis and results distribution tasks, with optional human intervention where needed for review and confirmation.</p>
<p><a href="https://www.globus.org/user-stories">User stories</a> and <a href="https://www.globus.org/usage-brief-library">usage briefs</a> provide multiple examples of how users have widely applied Globus to research data management challenges.</p>
<p>The University of Chicago develops and operates Globus and provides free file transfer for users engaged in non-profit research. Institutions can <a href="https://www.globus.org/why-subscribe">subscribe</a> to Globus to access additional features. Pricing is based on the research expenditures of the institution.</p>
</div>
<div id="basespace-sequence-hub" class="section level2 hasAnchor" number="6.7">
<h2><span class="header-section-number">6.7</span> BaseSpace Sequence Hub<a href="research-platforms.html#basespace-sequence-hub" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p><a href="https://basespace.illumina.com/">BaseSpace</a> is a platform that allows for data analysis of Illumina sequencing data and syncs easily with any Illumina sequencing machines that you might work with. There are many <a href="https://www.illumina.com/products/by-type/informatics-products/basespace-sequence-hub/apps.html">applications</a> available to help you with your genomics research. They offer a 30-day free trial.</p>
</div>
<div id="atlas.ti" class="section level2 hasAnchor" number="6.8">
<h2><span class="header-section-number">6.8</span> ATLAS.ti<a href="research-platforms.html#atlas.ti" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p><a href="https://atlasti.com/">ATLAS.ti</a> is designed particularly for qualitative analysis. You can use a variety of data types including video, audio, images, surveys, and social media data. A variety of tools, particularly for text data analysis, are provided for methods such as <a href="https://en.wikipedia.org/wiki/Sentiment_analysis">sentiment analysis</a>, which is the process of assigning a general tone or feeling to text and <a href="https://en.wikipedia.org/wiki/Named-entity_recognition">named-entity recognition</a>, which is the process of extracting certain characteristics from texts that are what is called a [named entity] or a real-world object - such as a person’s name or address. Such analyses can be helpful for understanding behaviors that might be associated with cancer risk. Although this type of analysis can be performed using R or Python among other coding languages, ATLAS.ti offers a nice graphical user interface to perform these types of analyses. Furthermore, ATLAS.ti offers a great deal of flexibility about such analyses, enabling users to easily incorporate different data types.</p>
<iframe src="https://downloads.atlasti.com/docs/branding/atlasti_brochure_v9_EN_interactive_202110.pdf" width="672" height="400px" data-external="1">
</iframe>
</div>
<div id="genepattern" class="section level2 hasAnchor" number="6.9">
<h2><span class="header-section-number">6.9</span> GenePattern<a href="research-platforms.html#genepattern" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p><a href="https://www.genepattern.org/">GenePattern</a> is similar to Galaxy in that it provides a web-based interface for genomic analyses. You can upload your own data, use workflows and pipelines form others and more!</p>
<p>See <a href="https://www.genepattern.org/user-guide">here</a> to access their user guide and <a href="https://notebook.genepattern.org/quickstart/">here</a> for a quick start guide to using <a href="https://notebook.genepattern.org/">GenePattern Notebook</a> which uses <a href="https://jupyter.org/">Jupyter Notebooks</a> and GenePattern analysis tools to easily create data analysis reports. Users can also publish and share their notebooks with collaborators or the field, as well as access other people’s notebooks that they can adapt for their own uses. See <a href="https://notebook.genepattern.org/library/">here</a> for a collection of available notebooks.</p>
</div>
<div id="xnat" class="section level2 hasAnchor" number="6.10">
<h2><span class="header-section-number">6.10</span> XNAT<a href="research-platforms.html#xnat" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p><a href="https://www.xnat.org/about/">XNAT</a> offers computing resources and tools for performing imaging analysis and for storing and sharing imaging data in a HIPAA complaint manner (more on that in the coming). Developed by the <a href="https://cnl.rc.fas.harvard.edu/">Bukner lab</a> previously at the Washington University and now at Harvard, it supports a variety of imaging data as well as other data types like clinical data. Some tools can be used with a graphical interface and others with the command-line. See <a href="https://wiki.xnat.org/documentation/case-studies">here</a> for example use cases. There is also a great deal of documentation available about how to use the tools and resources available at <a href="https://wiki.xnat.org/documentation" class="uri">https://wiki.xnat.org/documentation</a>.</p>
<iframe src="https://www.youtube.com/embed/ENk589mOkhI" width="100%" height="400px" data-external="1">
</iframe>
</div>
<div id="ohif" class="section level2 hasAnchor" number="6.11">
<h2><span class="header-section-number">6.11</span> OHIF<a href="research-platforms.html#ohif" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>The <a href="https://ohif.org">open health imaging foundation (OHIF)</a> is a web-based imaging analysis platform that is widely used, particularly for radiology analysis, but it also supports whole-slide microscopy image analysis. It was developed by <a href="https://ohif.org/team/">Gordon Harris</a> et al. and can be used for a variety of applications from cardiology to veterinary medicine. Check out these <a href="https://ohif.org/examples">example use cases</a> of OHIF. OHIF also provides thorough documentation with images and videos about how to use the image viewer and tools available.</p>
<p><img src="resources/images/06-General_Platforms_files/figure-html/1B4LwuvgA6aUopOHEAbES1Agjy7Ex2IpVAoUIoBFbsq0_gfd8d3f477a_159_2.png" alt="The OHIF image viewer (center) includes a tool bar for to modify the viewing and annotation of images (above center), a menu on the left to access images related to the current patient in other studies, and a menu on the right for making measurements and exporting measurement reports." width="100%" style="display: block; margin: auto;" /></p>
<p>For those interested, Gordon Harris and others are also working on a project called <a href="https://docs.cornerstonejs.org/">Cornerstone</a>, with the goal of providing software for others to display medical images in web browsers.</p>
</div>
<div id="prism" class="section level2 hasAnchor" number="6.12">
<h2><span class="header-section-number">6.12</span> PRISM<a href="research-platforms.html#prism" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>The Platform for Imaging in Precision Medicine, called PRISM, works behind the scenes in the Cancer Imaging Archive (TCIA) to allow users to work with the vast data available in TCIA, in terms of both imaging data and clinical information.</p>
<p>According to Fred Prior:</p>
<blockquote>
<p>It is designed to collect, curate and manage Radiology and Pathology images, clinical information associated with those images, annotations and image derived feature sets. PRISM is designed to run on a Kubernettes cluster, but all of the components are containerized so they can run stand-alone or in an alternate orchestration framework.</p>
</blockquote>
<p>See this <a href="https://ascopubs.org/doi/full/10.1200/CCI.20.00001">article</a> for more information.</p>
</div>
<div id="conclusion-4" class="section level2 hasAnchor" number="6.13">
<h2><span class="header-section-number">6.13</span> Conclusion<a href="research-platforms.html#conclusion-4" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>We hope that this chapter has given you some more perspective on the various computing options available designed for researchers like you. We also hope that you may have learned about some potential platforms that can help you to make your research faster and more flexible.</p>
<p>In conclusion, here are some of the major take-home messages:</p>
<ol style="list-style-type: decimal">
<li><p>Computing platforms are cyberinfrastructures that allow you to perform analyses with existing data and or allow you to upload, work with, and share your own data.</p></li>
<li><p>There are quite a few platforms that are well developed with in depth documentation to support research quite broadly and allow users to work with a diverse set of data.</p></li>
<li><p>There are also platforms that were designed for specific research needs.</p></li>
</ol>

</div>
</div>
<h3>References<a href="references.html#references" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-sciserver_2020" class="csl-entry">
Taghizadeh-Popp, M., J. W. Kim, G. Lemson, D. Medvedev, M. J. Raddick, A. S. Szalay, A. R. Thakar, et al. 2020. <span>“<span>SciServer</span>: <span>A</span> Science Platform for Astronomy and Beyond.”</span> <em>Astronomy and Computing</em> 33 (October): 100412. <a href="https://doi.org/10.1016/j.ascom.2020.100412">https://doi.org/10.1016/j.ascom.2020.100412</a>.
</div>
<div id="ref-talirz_materials_2020" class="csl-entry">
Talirz, Leopold, Snehal Kumbhar, Elsa Passaro, Aliaksandr V. Yakutovich, Valeria Granata, Fernando Gargiulo, Marco Borelli, et al. 2020. <span>“Materials <span>Cloud</span>, a Platform for Open Computational Science.”</span> <em>Scientific Data</em> 7 (1): 299. <a href="https://doi.org/10.1038/s41597-020-00637-5">https://doi.org/10.1038/s41597-020-00637-5</a>.
</div>
</div>
<hr>
<center>
<div class="container">
  <iframe class="responsive-iframe" src="https://c-savonen.shinyapps.io/widget-survey/?course_name=computing" style="width: 400px; height: 220px; overflow: auto;"></iframe>
</div>
  </div>
</center>

<hr>
<center>
  <div class="footer">
      All illustrations <a href="https://creativecommons.org/licenses/by/4.0/">CC-BY. </a>
      All other materials <a href= "https://creativecommons.org/licenses/by/4.0/"> CC-BY </a> unless noted otherwise.
  </div>
</center>
            </section>

          </div>
        </div>
      </div>
<a href="shared-computing-etiquette.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="computing-resource-decisions.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
